src/cc2olx/olx.py

```diff
@@ -245,9 +258,19 @@ def _fallback_olx_nodes(self) -> List["xml.dom.minidom.Element"]:
 
         return [html_node]
 
-    def _post_process(self, olx_node: "xml.dom.minidom.Element") -> None:
+    def _post_process(self, olx_node: "xml.dom.minidom.Element", idref: str) -> None:
         """
         Perform additional processing of the generated OLX node.
         """
         for post_processor in self._content_post_processors:
-            post_processor.process(olx_node)
+            try:
+                post_processor.process(olx_node)
```

Comment:
Question: Do we need to log successful post-processing like we log successful processing?